code/modules/projectiles/projectile.dm

```diff
@@ -63,6 +64,18 @@
 
 /obj/item/projectile/proc/Range()
 	range--
+	if(seeking_Angle_change && !QDELETED(original) && can_hit_target(original, permutated, TRUE))
+		Angle = SimplifyDegrees(Angle)
+		var/Angle_temp = Angle - 180
+		var/Angle_to_target = Get_Angle(src, original) - 180
+		var/Angle_difference = Angle_to_target - Angle_temp
+		if(SimplifyDegrees(Angle_temp) == SimplifyDegrees(Angle_to_target + 180) || SimplifyDegrees(Angle_temp + 180) == SimplifyDegrees(Angle_to_target)) //going directly opposite
+			Angle_difference = pick(seeking_Angle_change, -seeking_Angle_change)
+		else if(Angle_difference < -seeking_Angle_change)
+			Angle_difference = -seeking_Angle_change
+		else if(Angle_difference > seeking_Angle_change)
+			Angle_difference = seeking_Angle_change
+		Angle = SimplifyDegrees(Angle + Angle_difference)

```

Comment:
I need the projectile to pick a random direction if it's facing exactly the opposite angle from the target.
Also, if NORM_ROT() is supposed to normalize angles between 1 and 360 SimplifyDegrees() does that already in a much more obvious way; NORM_ROT is this; `#define NORM_ROT(rot) ((((rot % 360) + (rot - round(rot, 1))) > 0) ? ((rot % 360) + (rot - round(rot, 1))) : (((rot % 360) + (rot - round(rot, 1))) + 360))`